plugins/modules/rds_instance.py

```diff
@@ -1290,7 +1300,6 @@ def main():
         s3_ingestion_role_arn=dict(),
         s3_prefix=dict(),
         skip_final_snapshot=dict(type='bool', default=False),
-        snapshot_identifier=dict(),
```

Comment:
I guess this introduces a breaking change and we should probably handle accordingly.

Reply:
I didn't think so since I added it as an alias of `db_snapshot_identifier`

Reply:
I don't know if this needs to be handled in a different way. @tremble @markuman

Reply:
The reasoning behind this is restoring a db instance from a snapshot would fail using `snapshot_identifier`, since the parameter required for restoring an instance is `DBSnapshotIdentifier`, so that's why I added it as an alias of `db_snapshot_identifier`

Reply:
The key question IMO is "Could this break an existing playbook?"  I've tried poking through the logic and I can't spot any way that this would break an existing playbook...